apph.adoc

```diff
@@ -8,9 +7,9 @@
 === Point Data
 
 To represent data at scattered locations and times with no implied relationship among of coordinate positions, both data and coordinates must share the same (sample) instance dimension.  
-Because each feature contains only a single data element, there is no need for a separate element dimension.
-The representation of point features is a special, degenerate case of the standard four representations.
-The **`coordinates`** attribute is used on the data variables to unambiguously identify the relevant space and time auxiliary coordinate variables.
+Because each feature contains only a single data element, there is no need for a separate element dimension. 
+The representation of point features is a special, degenerate case of the standard four representations. 
+The `coordinates` attribute is used on the data variables to unambiguously identify the relevant space and time auxiliary coordinate variables.
```

Comment:
OK - we need a guide on bold/not bold! This would be for attribute names (e.g. standard_name), attribute values (e.g. air_temperature), and things which aren't attributes (e.g. humidity(i), which I picked out a few lines changes below.

I don't know what is best.

Reply:
I'd say... Let's address it by looking at [the style guidelines](https://github.com/cf-convention/cf-conventions/blob/main/CONTRIBUTING.md#text-styling) and potentially update across the doc after having that discussion. Otherwise this issue gets too big :)